### PR TITLE
Expose swagger-test-templates load-testing API

### PR DESCRIPTION
**Pull request by noahdietz**
@theganyo @mohsen1 how should the interface work? Based on the API in [swagger-test-templates](https://github.com/apigee-127/swagger-test-templates/blob/master/README.md) how should the specified paths+operations be passed through the CLI? 

Can I suggest passing the path to a JSON object/file that reflects the array of objects outlined in the API?

### CODE REVIEW

**Comment by noahdietz** [comment]
@theganyo @mohsen1 i put together a first draft of this option's implementation. Feedback welcome!

**Comment by noahdietz** [comment]
would it be better to require the file be named ```load-config.json``` instead of having them pass the file's path in the CLI?

**Comment by noahdietz** [comment]
also, this can't be merged until swagger-test-templates gets bumped to 1.2.0.

**Comment by theganyo** [comment]
@noahdietz Regarding your question: You could have it attempt to find the default file `load-config.json` if none is specified and then make the path optional. Then, if it doesn't find the file, it should emit an appropriate error message letting the user know what to do about that.

**Comment by noahdietz** [comment]
@theganyo thanks, that is a good idea. Checkout my implementation

File: lib/commands/project/project.js [hunk 1]
```diff
@@ -456,6 +457,17 @@ function testGenerate(directory, options, cb) {
             assertionFormat: assertionFormat
           };
 
+          // pass list of paths targeted for load testing
+          if (loadTesting) {
+            if ((typeof loadTesting) !== 'boolean' && fs.existsSync(path.join(directory, loadTesting))) {
+              config.loadTest = (JSON.parse(fs.readFileSync(path.join(directory, loadTesting)))).loadTargets;
```
Comment (mohsen1) [inline lib/commands/project/project.js]: If we're doing this in multiple places, shouldn't we make a helper function for it?

```
JSON.parse(fs.readFileSync(path.join(directory, "FILE NAME")));
```

**Comment by noahdietz** [comment]
@mohsen1 modularized that repeated code

**Comment by theganyo** [comment]
@noahdietz Is "Cannot find load testing configuration file" enough information for the user to fix this? I was thinking something a little more helpful if they don't specify a file at all...

**Comment by noahdietz** [comment]
So, if they don't specify a file but still use the ```-l``` flag, I look for ```load-config.json``` and if I don't find that I throw the error.

Should it read something like "Please specify a load test config or make a load-config.json file"?

**Comment by theganyo** [comment]
Maybe this? "Config file not found. Please specify a load test config or add load-config.json file to your project directory."

**Comment by noahdietz** [comment]
Sounds good

**Pull request merged**